test/ElasticSearchTest.php

```diff
@@ -257,6 +257,8 @@ public function testCreateIndexes_callIndexCreationBasedOnIndexOption(): void
             ->method('indices')
             ->willReturn($indexMock);
 
+        $this->sut->setOption('indexFiles', './vendor/oat-sa/lib-tao-elasticsearch/test/sample/testIndexes.conf.php');
```

Comment:
Should not we use relative path, since it is inside the library already?

Reply:
This is wrong approach in unit test. I just fix it but I'm not entirely sure if we should use other php file to load indexFiles. I would consider fix this as a tech debt in near future.

Reply:
Yes, I agree, but I am referring to the vendor path you add. Since you are inside the extension, can you use the relative path?

Reply:
I was trying but I had issues with this. If anybody can help me with this we can have a chat. I could also create temporary php file and require it.

Reply:
```
__DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'sample' . DIRECTORY_SEPARATOR . 'testIndexes.conf.php'
```
Maybe something like this?

Reply:
I found a way. I think this will be sufficient for you. :)